configure_machine/bootstrap_bevy_member_here.py

```diff
@@ -945,254 +983,267 @@ def display_introductory_text():
+        my_settings.setdefault('master_host',  False)  # assume this machine is NOT the VM host for the Master
+        if bevy == "local":
+            my_settings['master'] = True  # a masterless system is a master to itself
+            my_settings['projects_root'] = get_projects_directory()
+            get_additional_roots('file_roots')
+            get_additional_roots('pillar_roots')
+            default = my_settings.get('id', platform.node().split('.')[0])
+        elif interactive:
+            print('\n\nThis program can make this machine a simple workstation to join the bevy')
+            if platform.system() != 'Windows':
+                print('or a bevy salt-master (including cloud-master),')
+            if on_a_workstation:
+                print('or a Vagrant host, possibly hosting a bevy master.')
+            my_settings.setdefault('master', False)
+            if platform.system() != 'Windows':
+                my_settings['master'] = get_affirmation('Should this machine BE the master?', my_settings['master'])
+            if not my_settings['master'] and on_a_workstation:
+                my_settings['master_host'] = get_affirmation('Will the Bevy Master be a VM guest of this machine?',
+                                                             my_settings['master_host'])
+            my_settings['projects_root'] = get_projects_directory()
+            get_additional_roots('file_roots')
+            get_additional_roots('pillar_roots')
+            print()
+            first_id = get_salt_minion_id()
 
-    my_settings.setdefault('master_host',  False)  # assume this machine is NOT the VM host for the Master
-    if settings['bevy'] == "local":
-        my_settings['master'] = True  # a masterless system is a master to itself
-        get_additional_roots()
-        default = my_settings.get('id', platform.node().split('.')[0])
-    elif interactive:
-        print('\n\nThis program can make this machine a simple workstation to join the bevy')
-        if platform.system() != 'Windows':
-            print('or a bevy salt-master (including cloud-master),')
-        if on_a_workstation:
-            print('or a Vagrant host, possibly hosting a bevy master.')
-        my_settings.setdefault('master', False)
-        if platform.system() != 'Windows':
-            my_settings['master'] = get_affirmation('Should this machine BE the master?', my_settings['master'])
-        if not my_settings['master'] and on_a_workstation:
-            my_settings['master_host'] = get_affirmation('Will the Bevy Master be a VM guest of this machine?',
-                                                         my_settings['master_host'])
-        get_additional_roots()
+            if my_settings['master'] and not settings['bevy'] == "local":
+                print('NOTE: The Salt Node ID of the Bevy Master on itself should by "bevymaster".')

```

Comment:
Found a typo ("should be" instead of "should by"). Do I get a prize? :)

Reply:
Done.